### ISSUE TITLE
Error in Authentication_Score

### ISSUE THREAD

**Issue by renatolatorre** [NBISweden/aMeta#157]
Hi everyone,

I can't find my way around this error and would greatly appreciate your help.

```
Error in rule Authentication_Score:
    jobid: 1289
    input: results/MALT/MST2_055_m.trimmed.rma6, results/AUTHENTICATION/MST2_055_m/2483404/MaltExtract_output/log.txt, results/AUTHENTICATION/MST2_055_m/2483404/name_list.txt
    output: results/AUTHENTICATION/MST2_055_m/2483404/authentication_scores.txt
    log: logs/AUTHENTICATION_SCORE/MST2_055_m_2483404.log (check log file(s) for error details)
    conda-env: /aMETA/aMeta/.snakemake/conda/f662868e1efffa191f0ae26f948c8cee_
    shell:
        Rscript /Bioprograms/aMETA/aMeta/workflow/scripts/score.R results/MALT/MST2_055_m.trimmed.rma6 $(dirname results/AUTHENTICATION/MST2_055_m/2483404/MaltExtract_output/log.txt) results/AUTHENTICATION/MST2_055_m/2483404/name_list.txt $(dirname results/AUTHENTICATION/MST2_055_m/2483404/name_list.txt) &> logs/AUTHENTICATION_SCORE/MST2_055_m_2483404.log;
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

```

It happens with all taxid entering that part. Then the log contains the following:

```
Error in if (sum(df$V4 > 3)/dim(df)[1] > 0.1) { :
  missing value where TRUE/FALSE needed
Execution halted
```

Thanks in advance.

**Comment by ZoePochon** [comment]
Hi @renatolatorre !

I have two questions in order to see in which direction we could investigate. 
1. Do you use mobaxterm and have you opened the script plot.R with it? When mobaxterm saves files, it reformats them and then the quotes don't look the same etc and the code doesn't work anymore.
2. Could you check if the results/AUTHENTICATION/{sample}/{taxid}/PMDscores.txt were generated for the species for which the error arises? And do these files look fine?

**Comment by LeandroRitter** [comment]
@renatolatorre I believe (as @ZoePochon says) that there was not PMDscores.txt file generated for that taxid. It can also be that the file is there but empty, so division by dim(df)[1] diverges. Please check this. In any case, thanks for reporting, we should have a closer look at this soon.

**Comment by renatolatorre** [comment]
Hi, thanks for the replies. I don't use mobaxterm. I use Ubuntu on Windows to connect to a server and didn't try to open the R script before the errors. Also, at the time I checked that PMDscores.txt were generated for the taxid with errors and contained information inside.
During these days I managed to finish running aMETA by restarting it several times when the error arised. I think when restarting, by chance it started and finished with other jobs before going to that authentication step
